src/thor/bidirectional_astar.cc

```diff
@@ -337,14 +336,15 @@ inline bool BidirectionalAStar::ExpandInner(baldr::GraphReader& graphreader,
       dist = astarheuristic_forward_.GetDistance(t2->get_node_ll(meta.edge->endnode()));
     }
     edgelabels_reverse_.emplace_back(pred_idx, meta.edge_id, opp_edge_id, meta.edge, newcost,
-                                     sortcost, dist, mode_, transition_cost, thru,
-                                     (pred.closure_pruning() || !costing_->IsClosed(meta.edge, tile)),
+                                     sortcost, dist, mode_, transition_cost, not_thru_pruning,
+                                     pred.closure_pruning() || !(costing_->IsClosed(opp_edge, t2)),
```

Comment:
it's easy to miss: I changed to checking if `opp_edge` is closed when expanding reverse because that's the one we want to go on in the final path

Reply:
there's quite a few of these fixes sprinkled in this PR